src/data/list/big_operators.lean

```diff
@@ -451,6 +462,22 @@ le_antisymm (hl₂ ▸ single_le_prod hl₁ _ hx) (hl₁ x hx)
 lemma prod_eq_one [monoid M] {l : list M} (hl : ∀ (x ∈ l), x = (1 : M)) : l.prod = 1 :=
 trans (prod_eq_pow_card l 1 hl) (one_pow l.length)
 
+@[to_additive]
+lemma mem_ne_one_of_prod_ne_one [monoid M] {l : list M} (h : l.prod ≠ 1) :
+  ∃ (x ∈ l), x ≠ (1 : M) :=
+begin
+  by_contra hf, push_neg at hf,
+  exact h (prod_eq_one hf),
+end
+
+/-- If a product of integers is `-1`, then at least one factor must be `-1`. -/
+lemma int_prod_eq_neg_one {l : list ℤ} (h : l.prod = -1) : ∃ (x ∈ l), x = (-1 : ℤ) :=
```

Comment:
maybe
```suggestion
lemma exists_mem_eq_neg_one_of_prod_eq_neg_one {l : list ℤ} (h : l.prod = -1) : ∃ (x ∈ l), x = (-1 : ℤ) :=
```

Reply:
OK, it makes sense for this to be parallel to the other one.